frontend/demo/src/pages/Inbox/ConversationList/index.tsx

```diff
@@ -71,7 +71,7 @@ const ConversationList = (props: ConversationListProps) => {
 
     const items = conversations;
     const metadata = conversationsMetadata;
-    const hasMoreData = metadata.nextCursor && metadata.nextCursor.length > 0;
+    const hasMoreData = metadata.next_cursor && metadata.next_cursor.length > 0;
```

Comment:
I did a quick fix because I didn't want to complicate the code with a convoluted method (but I think the backend plans on changing the snake case fields to camelCase soon, so then we will fixed the types again :)